modules/ide-integration/src/main/resources/pga/airavata-php-gateway/app/config/app.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'debug' => false,
+    'debug' => true,
```

Comment:
@machristie I will revert it back to false. I was just thinking that whenever someone new tries to setup airavata they come across issues and since it is a development build we can keep debug = true so that they can find out the errors. Please let me know your views.

Reply:
fixed

Reply:
You know what, I forgot I was looking at the ide-integration PGA code 😄 . In that case I agree with your reasoning. Feel free to change it back.